test/Microsoft.TestPlatform.AcceptanceTests/TranslationLayerTests/CustomTestHostTests.cs

```diff
@@ -61,7 +61,7 @@ public void RunTestsWithCustomTestHostLauncherLaunchesTheProcessUsingTheProvided
     [TestMethod]
     [TestCategory("Windows-Review")]
     // [RunnerCompatibilityDataSource(BeforeFeature = Features.ATTACH_DEBUGGER_FLOW)]
-    [TestHostCompatibilityDataSource("net451", "netcoreapp2.1", "LegacyStable", BeforeFeature = Features.ATTACH_DEBUGGER_FLOW, DebugVSTestConsole = true)]
+    [TestHostCompatibilityDataSource(Net462TargetFramework, "netcoreapp2.1", "LegacyStable", BeforeFeature = Features.ATTACH_DEBUGGER_FLOW, DebugVSTestConsole = true)]
```

Comment:
I'd just replace this with the "net462" string, this should not he here at all if we are able to fix the debugger flow.